### ISSUE TITLE
Feature: Focus Files Panel and can be select by keyup and keydown

### ISSUE THREAD

**Issue by vankhuong** [typora/typora-issues#1001]
I really love feature focus to files panel and can be select by keyup and keydown, please can you update this convenient?

**Comment by zDEFz** [comment]
still a problem. Cannot use Typora without a mouse. Would love to see some efficiency like with vim/vimium.

**Comment by dayudodo** [comment]
2017 - 2021, so many years! why don't support plug-in? then we can define functions by outselves.

**Comment by Awkee** [comment]
So many people need rename/delete file shortcut ， but so many years ， we still can't use this feature ....
I regret buying this software...

**Comment by evnoj** [comment]
Chiming in to say that simple keyboard navigation of the outline and file panels are the only thing I'd need to switch to Typora from Obsidian, would love to see this feature.

**Comment by godcong** [comment]
Do that many people need the F2 renaming feature? Please add me, 😄.

**Comment by igorsantos07** [comment]
Why is it that hard to implement keyboard navigation on the sidebar? Or, at least, acknolwedge such an old story, with a bunch of duplicates already. You can open the sidebar with a keyboard shortcut, but then you can't do anything on it because the damn sidebar doesn't get focus after getting opened 🤡 

I was really considering buying it, but after all those complaints, it doesn't look like it's worth it. There's no roadmap either, and even finding the issues board was quite, quite hard on itself.

**Comment by jwhiting** [comment]
Flipping through a folder of markdown files with a keystroke is something of an unsolved problem on Mac OS. This would be a high value feature with just a low-effort keyboard shortcut.

By my research for Mac OS there really isn't a good solution here. QLMarkdown.app allows mac's quicklook to show markdown, but it's not editable, and it closes when you change focus. Obsidian, with a custom community plugin and custom shortcut to use that custom plugin, works too, but is a lot of work to setup and setup has to be done for every folder ("vault" in Obsidian parlance) you want to do it in. If you're like me and have markdown files in different places, the "vault" mentality breaks and it's not a solution.

In other words nobody is doing this and Typora could do it really well.
